waiter/test/waiter/scheduler_test.clj

```diff
@@ -18,6 +18,7 @@
             [clojure.core.async :as async]
             [clojure.string :as str]
             [clojure.test :refer :all]
+            [clojure.walk]
```

Comment:
Please import this as `[clojure.walk :as walk]`

Reply:
Done.